en/micro-integrator/docs/administer-and-observe/using-the-command-line-interface.md

```diff
@@ -1,486 +1,776 @@
 # Micro Integrator CLI
 
-The Micro Integrator CLI allows you to monitor the synapse artifacts (deployed in a specified Micro Integrator server) from the command line. The CLI is an alternative to the [management dashboard](../../administer-and-observe/working-with-monitoring-dashboard), which provides a graphical view of the deployed artifacts.
+The Micro Integrator CLI allows you to monitor the synapse artifacts (deployed in a specified Micro Integrator server) from the command line. The dashboard (as well as the [dashboard](../../administer-and-observe/working-with-monitoring-dashboard)) communicates with the management API of WSO2 Micro Integrator to function.
```

Comment:
"as well as the "  within bracket sound to break the flow while reading

Reply:
dashboard should be changed to CLI here. Applied the change. I think this is more clear now.